src/main/modules/oidc/index.ts

```diff
@@ -47,9 +47,9 @@ export class OidcMiddleware {
     server.use((req: Request, res: Response, next: NextFunction) => {
       if (req.session.user) {
         res.locals.isLoggedIn = true;
-
         return next();
       }
+      res.locals.isLoggedIn = false;
```

Comment:
why set a local if you are redirecting? do you mean to set the session?

Reply:
i can have a session value called loggedin but i dont see the point (right now) if we have the user one. I can assign it before the if statement so it checks if the user object is there but again, i dont see the point as res.locals only is available during the request.